app/Transformers/BibleVerseTransformer.php

```diff
@@ -52,6 +51,17 @@ public function transform($bible_verse)
             'verse_text'=> $bible_verse->verse_text,
             'fileset_id'=> $bible_verse->fileset_id,
             'fileset_set_type_code'=> $bible_verse->fileset_set_type_code,
+            'fileset_set_size_code'=> $bible_verse->fileset_set_size_code,
+            'bible_filesets' =>$bible_verse->fileset->bible->first()
+                ->filesetsWithoutMeta
+                ->map(function (BibleFileset $fileset) {
```

Comment:
why is the fileset data needed on each verse? this is redundant, since the same info will show on all verses

Reply:
@bradflood , It will be different because when we uses this endpoint and we filters by book and chapter, it will return more than one verses with the same book and chapter but with a different bible. So, the bible_filesets property will display the all filesets related with the bible, book and chapter.

You can see the following example: `/api/bibles/verses/eng/MAT/1`. `BookID=MAT and Chapter=1`
 
```json
       {
            "verse_start": 1,
            "verse_end": 1,
            "chapter": 1,
            "book_id": "MAT",
            "language_id": 6414,
            "bible_id": "EN1ESV",
            "verse_text": "The book of the genealogy of Jesus Christ, the son of David, the son of Abraham.",
            "fileset_id": "ENGESH",
            "fileset_set_type_code": "text_plain",
            "fileset_set_size_code": "C",
            "bible_filesets": [
                {
                    "id": "ENGESH",
                    "asset_id": "dbp-prod",
                    "set_type_code": "text_format",
                    "set_size_code": "C"
                },
                {
                    "id": "ENGESHN1DA-opus16",
                    "asset_id": "dbp-prod",
                    "set_type_code": "audio",
                    "set_size_code": "NT"
                },
                {
                    "id": "ENGESHN1DA",
                    "asset_id": "dbp-prod",
                    "set_type_code": "audio",
                    "set_size_code": "NT"
                },
                {
                    "id": "ENGESHP2DV",
                    "asset_id": "dbp-vid",
                    "set_type_code": "video_stream",
                    "set_size_code": "NTP"
                },
                {
                    "id": "ENGESHN1SA",
                    "asset_id": "dbp-prod",
                    "set_type_code": "audio_stream",
                    "set_size_code": "NT"
                }
            ]
        },
{
            "verse_start": 1,
            "verse_end": 1,
            "chapter": 1,
            "book_id": "MAT",
            "language_id": 6414,
            "bible_id": "ENGNIVA",
            "verse_text": "A record of the genealogy of Jesus Christ the son of David, the son of Abraham:",
            "fileset_id": "EN1NIV",
            "fileset_set_type_code": "text_plain",
            "fileset_set_size_code": "NT",
            "bible_filesets": [
                {
                    "id": "EN1NIVN2DA",
                    "asset_id": "dbp-prod",
                    "set_type_code": "audio_drama",
                    "set_size_code": "NT"
                },
                {
                    "id": "EN1NIVN1DA",
                    "asset_id": "dbp-prod",
                    "set_type_code": "audio",
                    "set_size_code": "NT"
                },
                {
                    "id": "EN1NIVN2SA",
                    "asset_id": "dbp-prod",
                    "set_type_code": "audio_drama_stream",
                    "set_size_code": "NT"
                },
                {
                    "id": "EN1NIVN1SA",
                    "asset_id": "dbp-prod",
                    "set_type_code": "audio_stream",
                    "set_size_code": "NT"
                }
            ]
        },
```